### ISSUE TITLE
The new Unknown type 57 message

### ISSUE THREAD

**Issue by suntong** [wechaty/wechaty-puppet-padplus#306]
**Is your feature request related to a problem? Please describe.**

A clear and concise description of what the problem is:

The quoted message was fine to me previously, as there is no special treatment necessary for them, but now, the quoted message is looking like this:

```xml
<?xml version="1.0"?>
<msg>
	<appmsg appid="" sdkver="0">
		<title>这是另一个测试</title>
		<des />
		<username />
		<action>view</action>
		<type>57</type>
		<showtype>0</showtype>
		<content />
		<url />
		<lowurl />
		<dataurl />
		<lowdataurl />
		<contentattr>0</contentattr>
		<streamvideo>
			<streamvideourl />
			<streamvideototaltime>0</streamvideototaltime>
			<streamvideotitle />
			<streamvideowording />
			<streamvideoweburl />
			<streamvideothumburl />
			<streamvideoaduxinfo />
			<streamvideopublishid />
		</streamvideo>
		<canvasPageItem>
			<canvasPageXml><![CDATA[]]></canvasPageXml>
		</canvasPageItem>
		<refermsg>
			<type>1</type>
			<svrid>4073350879766311324</svrid>
			<fromusr>496278083@chatroom</fromusr>
			<chatusr>wxid_dt093obz6v22</chatusr>
			<displayname>quoted from name</displayname>
			<msgsource />
			<content>这是一个测试</content>
		</refermsg>
		<appattach>
			<totallen>0</totallen>
			<attachid />
			<cdnattachurl />
			<emoticonmd5 />
			<aeskey />
			<fileext />
			<islargefilemsg>0</islargefilemsg>
		</appattach>
		<extinfo />
		<androidsource>0</androidsource>
		<thumburl />
		<mediatagname />
		<messageaction><![CDATA[]]></messageaction>
		<messageext><![CDATA[]]></messageext>
		<emoticongift>
			<packageflag>0</packageflag>
			<packageid />
		</emoticongift>
		<emoticonshared>
			<packageflag>0</packageflag>
			<packageid />
		</emoticonshared>
		<designershared>
			<designeruin>0</designeruin>
			<designername>null</designername>
			<designerrediretcturl>null</designerrediretcturl>
		</designershared>
		<emotionpageshared>
			<tid>0</tid>
			<title>null</title>
			<desc>null</desc>
			<iconUrl>null</iconUrl>
			<secondUrl>null</secondUrl>
			<pageType>0</pageType>
		</emotionpageshared>
		<webviewshared>
			<shareUrlOriginal />
			<shareUrlOpen />
			<jsAppId />
			<publisherId />
		</webviewshared>
		<template_id />
		<md5 />
		<weappinfo>
			<username />
			<appid />
			<appservicetype>0</appservicetype>
			<videopageinfo>
				<thumbwidth>0</thumbwidth>
				<thumbheight>0</thumbheight>
				<fromopensdk>0</fromopensdk>
			</videopageinfo>
		</weappinfo>
		<statextstr />
		<finderFeed>
			<objectId>null</objectId>
			<objectNonceId>null</objectNonceId>
			<feedType>0</feedType>
			<nickname />
			<username />
			<avatar><![CDATA[null]]></avatar>
			<desc />
			<mediaCount>0</mediaCount>
			<mediaList />
		</finderFeed>
		<findernamecard>
			<username />
			<avatar><![CDATA[]]></avatar>
			<nickname />
			<auth_job />
			<auth_icon>0</auth_icon>
			<auth_icon_url />
		</findernamecard>
		<finderTopic>
			<topic />
			<topicType>0</topicType>
			<iconUrl><![CDATA[null]]></iconUrl>
			<desc />
		</finderTopic>
		<finderEndorsement>
			<scene><![CDATA[0]]></scene>
		</finderEndorsement>
		<directshare>0</directshare>
		<websearch>
			<rec_category>0</rec_category>
			<channelId>0</channelId>
		</websearch>
	</appmsg>
	<fromusername>wxid_dt093obz6v22</fromusername>
	<scene>0</scene>
	<appinfo>
		<version>1</version>
		<appname></appname>
	</appinfo>
	<commenturl></commenturl>
</msg>
```

And the type is now "`Unknown`" instead of previously it was just plain text. 

**Describe the solution you'd like**

The quoted message being handled/interpreted as normal message again. 

**Describe alternatives you've considered**
Can't think of any alternative solutions.

**Additional context**
N.A.

[enhancement]

**Comment by su-chang** [comment]
Thank you for you issue, we will work on it later.

**Comment by su-chang** [comment]
Please check it in `wechaty-puppet-padplus@0.7.40`, if everything as expect, please close this issue. 
Thank you.

**Comment by suntong** [comment]
Thanks for the swift fix. 

However, I upgraded wechaty-puppet-padplus to v0.7.40, but the `default.memory-card.json` is no longer able to save my login info any more. Every time I start my node program, I was asked to scan the QR code to login.

My phone is dying and cannot be charged any more at the moment, so I can't test it now and for the near future. Feel free to close it if you believe the problem has been fixed.
